seed/api/v2_1/views.py

```diff
@@ -122,3 +125,89 @@ def building_sync(self, request, pk):
         """
         return JsonResponse(
             {"status": "error", "message": "Not yet implemented. PK was {}".format(pk)})
+
+    @detail_route(methods=['PUT'])
+    def update_with_building_sync(self, request, pk):
```

Comment:
I just put this name here to avoid conflicts with the GET method that we still need to wrap up.  When both methods are working well, it is simple to put them together into a single detail route.

Reply:
sounds good.